test.scss

```diff
@@ -0,0 +1,6 @@
+$color: #FFFFFF;
+
+body {
+ color: #FFFFFF;
```

Comment:
Color literals such as 'FFFFFF' should only be used in variable declarations (no-color-literals)